src/client.rs

```diff
@@ -1478,6 +1528,18 @@ pub fn linux_amd64_resolver(manifests: &[ImageIndexEntry]) -> Option<String> {
         .map(|entry| entry.digest.clone())
 }
 
+/// A platform resolver that chooses the first windows/amd64 variant, if present
+pub fn windows_amd64_resolver(manifests: &[ImageIndexEntry]) -> Option<String> {
```

Comment:
Why do we need this public function? It doesn't seem to be used anywhere inside of the code

Reply:
same as `fn linux_amd64_resolver` but for windows containers